eu_dgc_v1_schema.yaml

```diff
@@ -37,18 +37,28 @@ properties:
           type: object
           required:
             - t
+            - c
             - i
           properties:
             t:
               title: Identifier type
               description: The type of identifier (viz. VS-2021-04-08)
                 PPN = Passport
-                NNxxx = national Person Identifier (ISO 3166-1 alpha-3 country code)
+                NN = National Person Identifier
                 CZ = Citizenship card
                 HC = Health Card number
-                etc.
-              example: PPN
               type: string
+              enum:
+                - PPN
+                - NN
+                - CZ
+                - HC
+              example: NN
+            c:
```

Comment:
I think that the issuing country should be optional. If not set the country set in the issuer claim of the CWT should be assumed. This will be the case in 99% of the cases I guess.

Reply:
Fixed.